vmm/src/device_manager.rs

```diff
@@ -2293,8 +2293,8 @@ impl DeviceManager {
                             Box::new(RawFileDisk::new(file)) as Box<dyn DiskFile>
                         }
                     } else {
-                        info!("Using synchronous RAW disk file");
-                        Box::new(RawFileDiskSync::new(file)) as Box<dyn DiskFile>
+                        info!("Using asynchronous RAW disk file (aio)");
+                        Box::new(RawFileDiskAio::new(file)) as Box<dyn DiskFile>
```

Comment:
Can you add a check for the support of aio on the host at runtime (like what's done from `io_uring_is_supported()` and keep the sync I/O as default?

Reply:
Done. Unfortunately, aio doesn't have a comprehensive probing system like io_uring, but the check I added should be sufficient to detect cases where aio has been explicitly compiled out of the kernel.